### ISSUE TITLE
Where did I set my ckeys?

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#253]

**Comment by woothie** [comment]
It is. Have someone test it.

**Comment by woothie** [comment]
Regardless, as pointed out over skype, we should instead focus on fixing the fact that admins can't ban ckeys. Let's fix the admin tools before tinkering, yes?

**Comment by Davnik** [comment]
No. Setting ckeys via vars requires +DEBUG, and as has been pointed out, over-embellished, and generally discussed for many months, this is unnecessary, and all admins should be able to perform this necessary procedure for banning players who either never left the lobby, or ghosted and can no longer be found, or are being banned in a round they are not in, among other issues. We are not going to wait six months to have the right variety of developers to create an HTML-encoded menu for banning specific ckeys and reviewing those bans, nor are we going to waste days trying to fix Unban-Panel, nor are we ALL going to get phpmyadmin, and so on and so on and so on and so on. There are far more complications with any option other than to fix the immediate issue here. Simply put, it requires one var too many to perform this core admin duty. Don't cause unnecessary bureaucratic slowdown.

**Comment by Davnik** [comment]
Also, it isn't, I just tested it. http://puu.sh/kAADB/35ac5e102e.png

**Comment by Mewchild** [comment]
@woothie @aztectornado
